README.rst

```diff
@@ -101,10 +101,12 @@ Finally (graceful) restart httpd daemon.
 Koji builder
 ~~~~~~~~~~~~
 
-`osbs <https://github.com/DBuildService/osbs>`_ package is required. In Fedora it
-is part of official repositories. Additionally you'll need to modify
-`/etc/osbs.conf` with addresses to OpenShift buildystem instance and registry.
-Follow osbs documentation if you find any.
+The Koji builder plugin requires the `osbs-client
+<https://github.com/projectatomic/osbs-client>`_ package. In Fedora it is part
+of the official repositories. Additionally you'll need to modify
+`/etc/osbs.conf` with the addresses to your OpenShift buildystem instance and
+registry. Follow the `osbs documentation
+<https://github.com/projectatomic/osbs-client/tree/master/docs>`_.
```

Comment:
Actually a better landing site might be: https://osbs.readthedocs.org/

Reply:
Thanks Tim. I've amended this PR with your suggested URL.